jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/awsrules/utils/PacmanUtils.java

```diff
@@ -4162,6 +4179,30 @@ public static String getQualysVulnerabilitiesDetails(JsonArray vulnerabilities)
 
     }
 
+    public static String getCrowdstrikeVulnerabilitiesDetails(JsonArray vulnerabilities) {

```

Comment:
We must refactor our policies and Not expose specific plugin names.

Instead, we need a common pattern

Reply:
got it